packages/fuel-gauge/src/predicate/predicate-configurables.test.ts

```diff
@@ -198,6 +198,50 @@ describe('Predicate', () => {
       await assertBalance(destination, amountToTransfer, provider.getBaseAssetId());
     });
 
+    it('calls a predicate with partial configurables being set', async () => {
+      using launched = await launchTestNode();
+
+      const {
+        provider,
+        wallets: [wallet],
+      } = launched;
+
+      const configurableConstants = {
+        ADDRESS: getRandomB256(),
+      };
+
+      const amountToTransfer = 300;
+
+      const predicate = new Predicate({
```

Comment:
I think given we are testing out the typegen for this use case we should probably use the `createInstance` syntax to demonstrate the type system is accommodating the partiality.  e.g.

```ts
const predicate = PredicateWithConfigurableAbi__factory.createInstance(
  provider,
  [myFee, myAddress ],
  { FEE: 234 }
);
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2815/commits/0242f3bd6ae7a9bd9bf455ddfc07c6052113cc62